client/src/locales/en-US.json

```
@@ -21,7 +21,8 @@
         "incompatibleServer": "Your application and the server are not compatible. Online features will not be available. Please check that your application is up-to-date, and if so, contact an administrator.",
         "clientRevoked": "You have been revoked from this organization. You can still see some of the data you had access to, but most features of Parsec will not work.",
         "expiredOrganization": "This organization has expired. Please contact an administrator.",
-        "unexpected": "An unexpected error occurred. Please try again."
+        "unexpected": "An unexpected error occurred. Please try again.",
+        "smallDisplayWarning": "This website is not (yet) made to run on small displays. We strongly advice you to only use this website on a larger display."
```

Instead of the generic "website" I would use Parsec (or "Parsec web", "Parsec web app", etc.). 
```suggestion
        "smallDisplayWarning": "Parsec is not yet fully compatible with small displays. We strongly recommend that you use a larger display to access it."
```